lib/util.js

```diff
@@ -460,9 +461,9 @@ exports.chooseRandom = chooseRandom;
  * @param {Number} currentSleepTime 
  * @returns {Boolean} return jitter.
  */
-function getNextSleepTime (numofRetries, base, currentSleepTime) {
+function getNextSleepTime (numofRetries,  currentSleepTime) {
```

Comment:
Hi, I would add a unit test that confirms that next sleep time is always greater than the previous one. Something similar to this: https://github.com/snowflakedb/gosnowflake/blob/28a31f07ee411ec526b04f346c2b7806a1adc677/retry_test.go#L552

Reply:
There is a failure scenario for this. If there is no logintimeout, this test will be reasonable. However, If there is a max timeout, and max timeout - totalTimeout (<- add all the previous timeouts) is less than  the last time and and greater than 0, the test will be failed in the last attempt. I can make a test without the last attempt, but I am not sure this will be reasonable.